### PR TITLE
Add exoflex to code scope

### PR DESCRIPTION
**Pull request by darcien**
Now we can use exoflex component directly in the live editor.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,18 +6,22 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "docusaurus deploy",
+    "postinstall": "patch-package"
```
Comment (oshimayoan) [inline package.json:10]: Isn't this suppose to be like this?
```suggestion
    "postinstall": "yarn patch-package"
```

File: package.json [hunk 2]
```diff
@@ -6,18 +6,22 @@
...
+    "core-js": "3",
     "exoflex": "^3.3.0",
+    "patch-package": "^6.2.1",
+    "postinstall-postinstall": "^2.1.0",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-native": "0.60.6",
-    "react-native-web": "^0.12.2"
+    "react-native-svg": "^12.0.3",
+    "react-native-web": "^0.11.0"
```
Comment (oshimayoan) [inline package.json:24]: Shall we `0.11.7` instead?
```suggestion
    "react-native-web": "^0.11.7"
```

File: package.json [hunk 3]
```diff
@@ -31,5 +35,15 @@
...
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-flow": "^7.9.0",
+    "@babel/preset-react": "^7.9.4",
+    "@babel/preset-typescript": "^7.9.0",
+    "@expo/webpack-config": "^0.11.14",
```
Comment (oshimayoan) [inline package.json:45]: I don't think we need this, don't we?

File: package.json (hunk 1 above)
Comment (darcien) [reply package.json:10]: Both are the same assuming the binaries in node modules is in the path.
I just match that one with the one from `patch-package` readme
```diff
 "scripts": {
+  "postinstall": "patch-package"
 }
```
https://github.com/ds300/patch-package#set-up

File: package.json (hunk 2 above)
Comment (darcien) [reply package.json:24]: It's the same since we use ^ right?

File: package.json (hunk 3 above)
Comment (oshimayoan) [reply package.json:45]: Shoot! It looks like I pointed 4 lines 😨 
What I mean is only the last line, `@expo/webpack-config`. I don't see we use it anywhere.

File: package.json (hunk 2 above)
Comment (darcien) [reply package.json:24]: Actually, I think it's better if we remove the caret and use specific version instead.

**Pull request merged**